.github/workflows/build.yml

```diff
@@ -51,22 +71,22 @@ jobs:
     runs-on: ubuntu-latest
     needs: [build]
     permissions:
-      contents: write # IMPORTANT: mandatory for making GitHub Releases
-      id-token: write # IMPORTANT: mandatory for sigstore
+      contents: write     # IMPORTANT: mandatory for making GitHub Releases
+      id-token: write     # IMPORTANT: mandatory for attestations
+      attestations: write # IMPORTANT: mandatory for attestations
```

Comment:
Check will pass when https://github.com/python-jsonschema/check-jsonschema/pull/429 is merged and released

Reply:
Done https://github.com/python-jsonschema/check-jsonschema/releases/tag/0.28.4